.travis.yml

```diff
@@ -8,9 +8,25 @@ before_script:
   - bundle install
   - psql -c 'create database theodinproject_test;' -U postgres
 script:
-  - bundle exec rake bundler:audit
-  - bundle exec rake db:migrate RAILS_ENV=test
+  - gem install bundler-audit
+  - bundle audit check --update --ignore CVE-2015-9284
+  - RAILS_ENV=test bundle exec rails db:schema:load
   - bundle exec rspec
-test:
-  adapter: postgresql
-  database: theodinproject_test
+deploy:
+  - &deploy
+    provider: heroku
+    strategy: git
+    edge: true
+    api_key:
+      secure: mQVwLlrNOjZKdoI4TlZAiWayX7FFc8bI9yglHxa1L7LMQUWKQdJIduCe78wHCAYc/vyu896+N9PIXSxoU6GHs1JomMEC7mxHZOrjyY6XIZQL4tAi2DHsKO1AkF4flTovNXZH/khn5XGLHkHgDNotYglHBsRkwKC7Gu/1s5rmRup5A2X9tvMfHJ8kFCK3DK0qoP6ZFxLR9FvBfEzc+erP1iinCcjPsf2eFrlUC9NJWVk6Thf0fXF185DkXs4F/te0DwHPrrzzw5Nb4fzX+id6kPmzffW0bTAKqIuNSO79Sau8t07w/zxacZk2QsGgPKPLgEj4oIY+afbM9VlETKL+KHwHDXlrYef0Wdrj60MQEpsnbl5EUvTmO+6Cmugf51mBjHbYRttFLTHF+4vEXg2NLIn+vtHnS43QA05OxIq0LyhP2pGJz1moE05bw9mksCQKPbs5e1VE2DEN8YN+SqKC8lDeoFxeyIHT6o6CZVdubmNXXLomhgyCbvIlYABc2aTgSs8NPw7Qum8TgNeHsBBqEzMXITpgWb/g5chF3V6n5JSZn/kZ42Fn09RyeN7TjwTUH7Xu69oTagfLLdL5p2LalzpYOJAdZ0ygDSSu7vj7aFIHEilNYZc5EHPISRVFhVi3P+M/bHrY/w1U0ZEQMeBlqwi/y8ZjnGcz3oEpHXuyrQw=
+    run:
+      # Ref: https://docs.travis-ci.com/user/deployment-v2/providers/heroku/#running-commands
+      - sleep 2
```

Comment:
The 2 seconds is chosen arbitrarily. Travis doesn't have a guideline on the sleep timing. We'll have to update if we find frequent build failures when trying to run `rails db:migrate`.

Reply:
We might be able to remove db:migrate from this script. I added a heroku release phase for migrating the db after a deploy earlier.
https://github.com/TheOdinProject/theodinproject/blob/master/Procfile

Reply:
Nice!